fastparquet/test/test_api.py

```diff
@@ -969,11 +969,12 @@ def test_multi_cat_split(tempdir):
 
 
 def test_multi(tempdir):
+    rng = np.random.default_rng()
```

Comment:
Please give this an explicit seed value, to ensure that we have the same values for every run.

Reply:
Here we are. My favorite number, it will necessarily help us to overcome this breathtaking PR... hopefully... ;)